### PR TITLE
Add structured-perf params to fpRerF.R

### PR DESCRIPTION
**Pull request by MrAE**

### CODE REVIEW

File: R-Project/R/fpRerF.R [hunk 1]
```diff
@@ -31,7 +38,7 @@
 
 
 fpRerF <-
-	function(X=NULL, Y=NULL,csvFileName=NULL, columnWithY=NULL, maxDepth = Inf, minParent=1, numTreesInForest=500, numCores=1,numTreeBins=NULL, forestType="binnedBaseRerF", nodeSizeToBin=NULL, nodeSizeBin=NULL,mtry=NULL, mtryMult=NULL,seed=sample(1:1000000,1)){
+	function(X=NULL, Y=NULL,csvFileName=NULL, columnWithY=NULL, maxDepth = Inf, minParent=1, numTreesInForest=500, numCores=1,numTreeBins=NULL, forestType="binnedBaseRerF", nodeSizeToBin=NULL, nodeSizeBin=NULL,mtry=NULL, mtryMult=NULL,seed=sample(1:1000000,1), methodToUse = 1, imageHeight = NULL, imageWidth = NULL, patchHeightMax = NULL, patchHeightMin = NULL, patchWidthMax = NULL, patchWidthMin = NULL){
```
Comment (falkben) [inline R-Project/R/fpRerF.R]: i think structured rerf should be able to be set in the "forest type" no?  then if that's set to "s-rerf" we set `methodToUse`...

File: R-Project/R/fpRerF.R [hunk 2]
```diff
@@ -78,6 +85,24 @@ fpRerF <-
 		forest_module$setParameterDouble("mtryMult",mtryMult)
 		}
 
+		if(forestType == "binnedBaseTern"){
```
Comment (falkben) [inline R-Project/R/fpRerF.R]: it isn't obvious to me why we require binnedBaseTern in order to use s-rerf...

Comment (falkben) [reply R-Project/R/fpRerF.R]: ... if binnedBaseRerF is specified, we set `methodToUse=1`.  if s-rerf -- `methodToUse=2`?

File: R-Project/man/fpRerF.Rd [hunk 3]
```diff
@@ -8,7 +8,9 @@ fpRerF(X = NULL, Y = NULL, csvFileName = NULL, columnWithY = NULL,
   maxDepth = Inf, minParent = 1, numTreesInForest = 500,
   numCores = 1, numTreeBins = NULL, forestType = "binnedBaseRerF",
   nodeSizeToBin = NULL, nodeSizeBin = NULL, mtry = NULL,
-  mtryMult = NULL, seed = sample(1:1e+06, 1))
+  mtryMult = NULL, seed = sample(1:1e+06, 1), methodToUse = 1,
```
Comment (falkben) [inline R-Project/man/fpRerF.Rd]: should the Readme.md file also have been changed?

**Review by falkben — CHANGES_REQUESTED** [review]
cool.  

i think we can make this more user friendly.

i'd love to see some tests to make sure that we can actually call it from `R`

File: R-Project/R/fpRerF.R (hunk 2 above)
Comment (MrAE) [reply R-Project/R/fpRerF.R]: Good idea.

File: R-Project/R/fpRerF.R (hunk 1 above)
Comment (MrAE) [reply R-Project/R/fpRerF.R]: Good point, that'd be much more clear.

**Comment by MrAE** [comment]
@falkben look like this was approved and I forgot about it?

I'm merging it because not having it caused me a couple hours worth of hunting down a segfault. 💥 
I am also adding an issue to write R specific S-RerF tests.

**Pull request merged**